devel/api.md

```diff
@@ -771,32 +775,32 @@ Example:
 ```python
 {
     Interface.KEY: [{
-        Interface.NAME: "linux-br0",
-        Interface.TYPE: LinuxBridge.TYPE,
-        Interface.STATE: InterfaceState.UP
-        LinuxBridge.CONFIG_SUBTREE: {
-            LinuxBridge.OPTIONS_SUBTREE: {
-                LinuxBridge.Options.GROUP_FORWARD_MASK: 0,
-                LinuxBridge.Options.MAC_AGEING_TIME: 300,
-                LinuxBridge.Options.MULTICAST_SNOOPING: true,
-                LinuxBridge.STP_SUBTREE: {
-                    LinuxBridge.STP.ENABLED: true,
-                    LinuxBridge.STP.FORWARD_DELAY: 15,
-                    LinuxBridge.STP.HELLO_TIME: 2,
-                    LinuxBridge.STP.MAX_AGE: 20,
-                    LinuxBridge.STP.PRIORITY: 32768
-                }
-            },
-            PORT_SUBTREE: [
-                {
-                    LinuxBridge.Port.NAME: "eth1",
-                    LinuxBridge.Port.STP_PRIORITY: 32,
-                    LinuxBridge.Port.STP_HAIRPIN_MODE: false,
-                    LinuxBridge.Port.STP_PATH_COST: 100
-                }
-            ]
-        }
-    }]
+            Interface.NAME: "linux-br0",
+            Interface.TYPE: LinuxBridge.TYPE,
+            Interface.STATE: InterfaceState.UP,
+            LinuxBridge.CONFIG_SUBTREE: {
+                LinuxBridge.OPTIONS_SUBTREE: {
+                    LinuxBridge.Options.GROUP_FORWARD_MASK: 0,
+                    LinuxBridge.Options.MAC_AGEING_TIME: 300,
+                    LinuxBridge.Options.MULTICAST_SNOOPING: True,
+                    LinuxBridge.STP_SUBTREE: {
+                        LinuxBridge.STP.ENABLED: True,
+                        LinuxBridge.STP.FORWARD_DELAY: 15,
+                        LinuxBridge.STP.HELLO_TIME: 2,
+                        LinuxBridge.STP.MAX_AGE: 20,
+                        LinuxBridge.STP.PRIORITY: 32768
+                    }
+                },
+                LinuxBridge.PORT_SUBTREE: [
+                    {
+                        LinuxBridge.Port.NAME: "eth1",
+                        LinuxBridge.Port.STP_PRIORITY: 32,
+                        LinuxBridge.Port.STP_HAIRPIN_MODE: False,
+                        LinuxBridge.Port.STP_PATH_COST: 100
+                    }
+                ]
+            }
+        }]
```

Comment:
These extra tabs are wrong. Please remove them. As you can see, each indentation is one tab size. Thanks!

Reply:
Done! Thank you :)